### PR TITLE
refactor: split FixedLists.json into multiple files

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

As in title

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/FixedLists/EPOReasons.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[  {
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/EPOReasons.json]: Fix indentation

File: ccd-definition/FixedLists/Consent.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[  {
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/Consent.json]: Fix indentation

File: ccd-definition/FixedLists/EmergencyProtectionOrderDirectionType.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+[  {
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/EmergencyProtectionOrderDirectionType.json]: Fix indentation

File: ccd-definition/FixedLists/EmergencyProtectionOrderType.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+[  {
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/EmergencyProtectionOrderType.json]: Fix indentation

File: ccd-definition/FixedLists/AllocationProposalList.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
+[  {
```
Comment (geek-soft) [inline ccd-definition/FixedLists/AllocationProposalList.json]: Fix formatting - please.

File: ccd-definition/FixedLists/LivingSituationList.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
+[
```
Comment (swalker125) [inline ccd-definition/FixedLists/LivingSituationList.json]: the listElementCode in this 🤢

File: ccd-definition/FixedLists/RelevantProceedings.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
+[
```
Comment (swalker125) [inline ccd-definition/FixedLists/RelevantProceedings.json]: would be nice to refactor these yes / no / don't know lists into one

File: ccd-definition/FixedLists/HearingTypeFixedList.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeFixedList",
+    "ListElementCode": "Standard case management hearing",
+    "ListElement": "Standard case management hearing",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeFixedList",
+    "ListElementCode": "Urgent preliminary case management heari",
```
Comment (lewisbirks) [inline ccd-definition/FixedLists/HearingTypeFixedList.json]: Sentence isn't finished

Comment (swalker125) [reply ccd-definition/FixedLists/HearingTypeFixedList.json]: ListElementCode 30 character limit 🤷‍♂

Comment (lewisbirks) [reply ccd-definition/FixedLists/HearingTypeFixedList.json]: This is 40 characters 😕

File: ccd-definition/FixedLists/AllocationProposalList.json (hunk 5 above)
Comment (akomisarek) [reply ccd-definition/FixedLists/AllocationProposalList.json]: Sorry, it's now fixed!

**Pull request merged**